DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -52,6 +79,10 @@ void DDFilteredView::mergedSpecifics(DDSpecParRefs const& specs) {
 }
 
 bool DDFilteredView::firstChild() {
+  if (it_.empty()) {
```

Comment:
@cvuosalo - there is always an Iterator present by construction.

Reply:
It looks like the `up()` method could empty `it_`, and then `firstChild()` would cause a crash without this check.